library/bignum.c

```diff
@@ -1328,7 +1328,8 @@ int mbedtls_mpi_add_abs( mbedtls_mpi *X, const mbedtls_mpi *A, const mbedtls_mpi
 }
 
 /*
- * Helper for mbedtls_mpi subtraction
+ * Helper for mbedtls_mpi subtraction:
+ * d -= s where d and s have the same size and d >= s.
  */
 static void mpi_sub_hlp( size_t n,
                          const mbedtls_mpi_uint *s,
```

Comment:
Preexisting: the order of arguments is confusing: `mpi_sub_hlp(n, s, d)` calculates `d - s` (swapped order) and puts the result into `d` (even though we normally put the result before input-only parameters).

Reply:
I swapped the argument. It makes no difference to the behavior, but it's easier to read.